docs/developer-docs/latest/developer-resources/database-apis-reference/entity-service/filter.md

```diff
@@ -88,7 +88,7 @@ const entries = await strapi.entityService.findMany('api::article.article', {
 `$not` can be used:
 
 - as a logical operator (e.g. in `filters: { $not: { // conditions… }}`)
-- or [as an attribute operator](#not-2) (e.g. in `filters: { attribute-name: $not: { … } }`).
+- or [as an attribute operator](#attribute-operators) (e.g. in `filters: { attribute-name: $not: { … } }`).
```

Comment:
I would recommend you keep the original link, as the intent was to explain that `not` can be used either as a logical operator or as an attribute operator.

(I know this is detected as a missing link by the `check-links` script but this is a false positive. You can use the `checklinksIgnoredFiles` array in VuePress' `config.js` if you want to ignore this file :-)